src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -537,5 +531,38 @@ private static Set<File> toggleFiles(File folder, boolean enable) throws IOExcep
 
 		return changed;
 	}
-	
+
+	private static void notifyOperators(String message, CommandSender sender, RedirectingLogHandler logHandler) {
+		if (SkriptConfig.sendReloadingInfoToOps.value()) {
+			Bukkit.getOnlinePlayers().stream()
+				.filter(player -> player.isOp() && player.hasPermission("skript.reloadnotify") && !player.equals(sender))
```

Comment:
should this really check for isOp? Just checking for the permission would be more flexible for the user.